frontend/src/citizen-frontend/async-rendering.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Result } from 'lib-common/api'
+import {
+  genericUnwrapResult,
+  UnwrapResultProps
+} from 'lib-components/async-rendering'
+import { useTranslation } from './localization'
+
+export function UnwrapResult<T>(props: UnwrapResultProps<T>) {
+  const t = useTranslation()
+  return genericUnwrapResult({
+    ...props,
+    failureMessage: t.common.errors.genericGetError
+  })
+}
+
+export interface RenderResultOptions<T> {
+  result: Result<T>
+  loading?: () => React.ReactElement | null
+  failure?: () => React.ReactElement | null
+}
```

Comment:
These interfaces are not used anywhere.

Reply:
Ah, true! I first tried to use `renderResult` for the case where loading and failure rendering can be customized, but it was not good.